src/main/scala/com.snowplowanalytics/weather/providers/openweather/Client.scala

```diff
@@ -28,25 +29,23 @@ import Responses._
 import Requests._
 
 /**
- * Base client trait with defined client methods such a `historyById`, `historyByName`
+ * Base client trait with defined client methods such as `historyById`, `historyByName`
  * common for subclasses
  *
- * @tparam Response response wrapper for `Client` subclass, such as `Future`
+ * @tparam F response wrapper for `Client` subclass, such as cats `IO`
```

Comment:
I think "effect type" would be enough: as the API stands there is no constraint on `F` so no need to mention `IO`.